XPetra: build failure

just updated trilinos and in a clean build am getting the following error in xpetra. Any suggestions?

@trilinos/xpetra 
@lucbv 

```
In file included from /ascldap/users/rppawlo/Trilinos/packages/xpetra/src/MultiVector/Xpetra_EpetraIntMultiVector.cpp:46:0:
/ascldap/users/rppawlo/Trilinos/packages/xpetra/src/MultiVector/Xpetra_EpetraIntMultiVector.hpp: In member function ‘virtual Kokkos::DualView<int**, Kokkos::LayoutStride, Kokkos::OpenMP, Kokkos::MemoryTraits<1> >::t_host_um Xpetra::EpetraIntMultiVectorT<int, Kokkos::Compat::KokkosDeviceWrapperNode<Kokkos::OpenMP> >::getHostLocalView() const’:
/ascldap/users/rppawlo/Trilinos/packages/xpetra/src/MultiVector/Xpetra_EpetraIntMultiVector.hpp:747:32: error: no matching function for call to ‘Epetra_IntMultiVector::ExtractView(int**)’
         vec_->ExtractView(&data);
                                ^
In file included from /ascldap/users/rppawlo/Trilinos/packages/xpetra/src/MultiVector/Xpetra_EpetraIntMultiVector.hpp:57:0,
                 from /ascldap/users/rppawlo/Trilinos/packages/xpetra/src/MultiVector/Xpetra_EpetraIntMultiVector.cpp:46:
/ascldap/users/rppawlo/Trilinos/packages/epetra/src/Epetra_IntMultiVector.h:560:7: note: candidate: int Epetra_IntMultiVector::ExtractView(int**, int*) const
   int ExtractView(int **A, int *MyLDA) const;
       ^~~~~~~~~~~
/ascldap/users/rppawlo/Trilinos/packages/epetra/src/Epetra_IntMultiVector.h:560:7: note:   candidate expects 2 arguments, 1 provided
/ascldap/users/rppawlo/Trilinos/packages/epetra/src/Epetra_IntMultiVector.h:573:7: note: candidate: int Epetra_IntMultiVector::ExtractView(int***) const
   int ExtractView(int ***ArrayOfPointers) const;
       ^~~~~~~~~~~
/ascldap/users/rppawlo/Trilinos/packages/epetra/src/Epetra_IntMultiVector.h:573:7: note:   no known conversion for argument 1 from ‘int**’ to ‘int***’
In file included from /ascldap/users/rppawlo/Trilinos/packages/xpetra/src/MultiVector/Xpetra_EpetraIntMultiVector.cpp:46:0:
/ascldap/users/rppawlo/Trilinos/packages/xpetra/src/MultiVector/Xpetra_EpetraIntMultiVector.hpp: In member function ‘virtual Kokkos::DualView<int**, Kokkos::LayoutStride, Kokkos::OpenMP, Kokkos::MemoryTraits<1> >::t_host_um Xpetra::EpetraIntMultiVectorT<long long int, Kokkos::Compat::KokkosDeviceWrapperNode<Kokkos::OpenMP> >::getHostLocalView() const’:
/ascldap/users/rppawlo/Trilinos/packages/xpetra/src/MultiVector/Xpetra_EpetraIntMultiVector.hpp:1185:32: error: no matching function for call to ‘Epetra_IntMultiVector::ExtractView(int**)’
         vec_->ExtractView(&data);
                                ^
In file included from /ascldap/users/rppawlo/Trilinos/packages/xpetra/src/MultiVector/Xpetra_EpetraIntMultiVector.hpp:57:0,
                 from /ascldap/users/rppawlo/Trilinos/packages/xpetra/src/MultiVector/Xpetra_EpetraIntMultiVector.cpp:46:
/ascldap/users/rppawlo/Trilinos/packages/epetra/src/Epetra_IntMultiVector.h:560:7: note: candidate: int Epetra_IntMultiVector::ExtractView(int**, int*) const
   int ExtractView(int **A, int *MyLDA) const;
       ^~~~~~~~~~~
/ascldap/users/rppawlo/Trilinos/packages/epetra/src/Epetra_IntMultiVector.h:560:7: note:   candidate expects 2 arguments, 1 provided
/ascldap/users/rppawlo/Trilinos/packages/epetra/src/Epetra_IntMultiVector.h:573:7: note: candidate: int Epetra_IntMultiVector::ExtractView(int***) const
   int ExtractView(int ***ArrayOfPointers) const;
       ^~~~~~~~~~~
/ascldap/users/rppawlo/Trilinos/packages/epetra/src/Epetra_IntMultiVector.h:573:7: note:   no known conversion for argument 1 from ‘int**’ to ‘int***’
[1502/5848] Building CXX object packages/tpetra/core/src/CMakeFiles/tpetra.dir/Tpetra_CrsMatrix_INT_INT_INT_OPENMP.cpp.o

```

--- a/packages/xpetra/src/MultiVector/Xpetra_EpetraIntMultiVector.hpp
+++ b/packages/xpetra/src/MultiVector/Xpetra_EpetraIntMultiVector.hpp
@@ -349,16 +349,6 @@ const Epetra_IntMultiVector & toEpetra(const MultiVector<int, int, GlobalOrdinal
       TEUCHOS_UNREACHABLE_RETURN(ret);
     }
 
-    /// \brief Return an unmanaged non-const view of the local data on a specific device.
-    /// \tparam TargetDeviceType The Kokkos Device type whose data to return.
-    ///
-    /// \warning DO NOT USE THIS FUNCTION! There is no reason why you are working directly
-    ///          with the Xpetra::EpetraIntVector object. To write a code which is independent
-    ///          from the underlying linear algebra package you should always use the abstract class,
-    ///          i.e. Xpetra::Vector!
-    ///
-    /// \warning Be aware that the view on the vector data is non-persisting, i.e.
-    ///          only valid as long as the vector does not run of scope!
     template<class TargetDeviceType>
     typename Kokkos::Impl::if_c<
       Kokkos::Impl::is_same<
@@ -744,11 +734,13 @@ const Epetra_IntMultiVector & toEpetra(const MultiVector<int, int, GlobalOrdinal
 
         // access Epetra vector data
         int* data = NULL;
-        vec_->ExtractView(&data);
+        int myLDA;
+        vec_->ExtractView(&data, &myLDA);
         int localLength = vec_->MyLength();
+        int numVectors  = vec_->NumVectors();
 
         // create view
-        epetra_view_type test = epetra_view_type(data, localLength,1);
+        epetra_view_type test = epetra_view_type(data, localLength, numVectors);
         typename dual_view_type::t_host_um ret = subview(test, Kokkos::ALL(), Kokkos::ALL());
 
         return ret;
@@ -1182,11 +1174,13 @@ const Epetra_IntMultiVector & toEpetra(const MultiVector<int, int, GlobalOrdinal
 
         // access Epetra vector data
         int* data = NULL;
-        vec_->ExtractView(&data);
+        int myLDA;
+        vec_->ExtractView(&data, &myLDA);
         int localLength = vec_->MyLength();
+        int numVectors  = vec_->NumVectors();
 
         // create view
-        epetra_view_type test = epetra_view_type(data, localLength, 1);
+        epetra_view_type test = epetra_view_type(data, localLength, numVectors);
         typename dual_view_type::t_host_um ret = subview(test, Kokkos::ALL(), Kokkos::ALL());
 
         return ret;

